torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5198,7 +5217,6 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     OpInfo('logsumexp',
```

Comment:
Interesting, so this does support complex autograd now? That's awesome

Reply:
well I dont see why that flag was needed in the first place, b/c no `dtypes*` includes complex type.